src/platform/plugins/shared/metrics_experience/server/lib/fields/enrich_metric_fields.test.ts

```diff
@@ -24,6 +24,7 @@ const extractDimensionsMock = extractDimensions as jest.MockedFunction<typeof ex
 const msearchMock = jest.fn() as jest.MockedFunction<TracedElasticsearchClient['msearch']>;
 const esClientMock = { msearch: msearchMock } as unknown as TracedElasticsearchClient;
 const normalizeUnitMock = normalizeUnit as jest.MockedFunction<typeof normalizeUnit>;
+const timeRangeFixture: EpochTimeRange = { to: Date.now() - 300_000, from: Date.now() };
```

Comment:
Why is this `300_000` and not `300000` 🤔

Reply:
Nit: I would put the `from` value first and add a comment `5 minutes ago` after the `to`, something like: 
```suggestion
const timeRangeFixture: EpochTimeRange = { from: Date.now(), to: Date.now() - 300000 // 5 minutes ago  };
```
wdyt?